.github/workflows/dockerhub-release.yml

```diff
@@ -65,9 +65,12 @@ jobs:
         run: |
           sudo skopeo login -u ${{ env.docker_user }} --password=${{ env.docker_password }} https://${{ env.DH_IMAGE_REGISTRY }}
 
-      - name: 🐳 Sync images to Docker Hub
+      - name: 🐳 Sync latest tag of images to Docker Hub
         run: |
-          sudo skopeo sync \
-            --src yaml \
-            --dest docker \
-            .github/promote-images.yml ${{ env.DH_IMAGE_REGISTRY }}/gitpod
+          IMAGES=$(cat .github/promote-images.yml | yq '."europe-docker.pkg.dev/gitpod-artifacts/docker-dev"."images-by-tag-regex"|keys[]' -r)
+          for IMAGE in $IMAGES;
+          do
+            sudo skopeo copy \
+            docker://${{ env.GAR_IMAGE_REGISTRY }}/gitpod-artifacts/docker-dev/$IMAGE:latest \
+            docker://${{ env.DH_IMAGE_REGISTRY }}/gitpod/$IMAGE:latest
+          done
```

Comment:
How is this getting the latest image for adding the latest tag? It seems like it will copy each image, and then try adding the latest tag. Let's fix in a follow-on PR, for now we can proceed, and just won't run this related action till resolved.

Reply:
This step copies latest tag from GAR to DH. The latest tag already [exists in GAR.](https://github.com/gitpod-io/workspace-images/blob/f142b2b3175b02cd90a5278b57ed744406065f60/.github/workflows/push-main.yml#L115-L119)

Reply:
:+1: awesome, thank you! I just reread the code with a fresh pair of eyes, and see that now too. Thank yous!